plugins/chat/app/services/chat_message_destroyer.rb

```diff
@@ -15,6 +15,7 @@ def trash_message(message, actor)
     ChatMessage.transaction do
       message.trash!(actor)
       ChatMention.where(chat_message: message).destroy_all
+      DiscourseEvent.trigger(:chat_message_deleted, message, message.chat_channel, actor)
```

Comment:
Would prefer `chat_message_trashed` to confer the fact it's not actually deleted